src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/worker.py

```diff
@@ -19,7 +20,7 @@ class Worker(ImageDetector, ABC):
     method.
     """
 
-    def __init__(self, image_queue: Deque, sio, **kw):
+    def __init__(self, image_queue, sio, **kw):
```

Comment:
put there annotation for image_queue and sio please

Reply:
Added